student-work/assignment-8/TANISHA-KRISHNARAJ/tree.py

```
@@ -0,0 +1,9 @@
+def count_unbalanced(bt):
+  if (bt.right == None and bt.left == None) or (bt.right and bt.left) :
```

Ruff (0.14.3)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)